### ISSUE TITLE
In defi_events.csv include sign in profit/loss

### ISSUE THREAD

**Issue by LefterisJP** [rotki/rotki#2267]
## Problem Definition

At the export of defi_events.csv there is no sign in profit/loss like for other csv exports. Even though we do calculate it properly in the PnL report the exported csv has no way to get the same number by adding up the PnL, so this needs to change.

## Task

Add negative sign in the non profitable defi events:

```python
    def is_profitable(self) -> bool:
        return self in (
            DefiEventType.DSR_LOAN_GAIN,
            DefiEventType.AAVE_LOAN_INTEREST,
            DefiEventType.COMPOUND_LOAN_INTEREST,
            DefiEventType.COMPOUND_REWARDS,
            DefiEventType.COMPOUND_DEBT_REPAY,
            DefiEventType.YEARN_VAULTS_PNL,
            DefiEventType.ADEX_STAKE_PROFIT,
        )
```

**Comment by LefterisJP** [comment]
Fixed by #2275

**Comment by hjorthjort** [comment]
Holy sh*t yes!
